app/stores/FileOperationsStore.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import invariant from "invariant";
+import { orderBy } from "lodash";
+import { computed } from "mobx";
+import FileOperation from "models/FileOperation";
+import BaseStore from "./BaseStore";
+import RootStore from "./RootStore";
+import { client } from "utils/ApiClient";
+
+export default class FileOperationsStore extends BaseStore<FileOperation> {
+  actions = ["list", "delete"];
+
+  constructor(rootStore: RootStore) {
+    super(rootStore, FileOperation);
+  }
+
+  @computed
+  get exports(): FileOperation[] {
+    return Array.from(this.data.values()).reduce(
+      (acc, fileOp) => (fileOp.type === "export" ? [...acc, fileOp] : acc),
+      []
+    );
+  }
+
+  @computed
+  get orderedDataExports(): FileOperation[] {
+    return orderBy(this.exports, "createdAt", "desc");
+  }
+
+  async viewExport(fileOperationId: string) {
+    const res = await client.post("/fileOperations.redirect", {
+      id: fileOperationId,
+    });
+    invariant(res && res.data, "Data should be available");
+    window.open(res.data, "_blank");
```

Comment:
While redirecting, AWS was rejecting the request because the browser included a few extra headers and bearer tokens (`Only one type of authentication should be present` was the error). 

How would you circumvent this issue?

Reply:
You should be able to just make this a plain link something like `a href="/fileOperations.redirect?id=<>"`, perhaps that will resolve?

Reply:
Works ✔️